packages/core/src/helpers/ast.ts

```diff
@@ -0,0 +1,26 @@
+import type * as postcss from 'postcss';
+
+// ToDo check empty file (position 0)
+export function getAstNodeAt(
+    root: postcss.Root,
+    offset: number
+): [node: postcss.AnyNode, offsetInNode: number] {
+    let closestNode: postcss.AnyNode = root;
+    let offsetInNode = 0;
+
+    root.walk((node) => {
+        if (node.source?.start && node.source?.end) {
```

Comment:
Do we care about zeros? What is this check for see that you have source?

Reply:
`start` and `end` are objects and cannot be 0 (they contain, line, column and offset).